submissions/src/test/kotlin/SubmissionControllerIntegrationTest.kt

```diff
@@ -128,6 +128,6 @@ class SubmissionControllerIntegrationTest {
         tableClient.listEntities().iterator().forEachRemaining { entities.add(it) }
         assertEquals(1, entities.size)
         val tableEntity = entities[0]
-        assertEquals("Received", tableEntity.getProperty("status"))
+        assertEquals("Received", tableEntity.getProperty("RowKey"))
```

Comment:
This one is weird. The table in the azure storage is nosql. So it operates on entities. Each entity must have a partition key and a row key. The azure library forces us to use "PartitionKey" and "RowKey" as the field names. This PR includes a change to that table (`submissions`). Previously that table had reportID for "PartitionKey" and "RowKey" then separate fields for status, reportID, and datetime. But after learning more about `PartitionKey` and `RowKey` it made sense to put reportID in `PartitionKey` and status in `RowKey` and there was already an auto created field for timestamp so I dropped the datetime field too.

So the change on this line is needed because I removed the `status` field and moved that data to `RowKey`. I wish we could specify the names of those two fields but the library is blocking that.

Reply:
Makes sense, thanks for explaining those Azure semantics 🙏 .